blueflood-core/src/main/java/com/rackspacecloud/blueflood/utils/EventModuleLoader.java

```diff
@@ -0,0 +1,45 @@
+package com.rackspacecloud.blueflood.utils;
+
+import com.rackspacecloud.blueflood.io.GenericElasticSearchIO;
+import com.rackspacecloud.blueflood.service.Configuration;
+import com.rackspacecloud.blueflood.service.CoreConfig;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.List;
+
+public class EventModuleLoader {
+    private static final Logger log = LoggerFactory.getLogger(EventModuleLoader.class);
+    private static GenericElasticSearchIO instance = null;
+
+    public static GenericElasticSearchIO getInstance() {
+        return instance;
+    }
+
+    public static synchronized void loadEventModule() {
```

Comment:
If you pull the latest changes from rackerlabs/master you will see that @chinmay-gupte has already added an ElasticSearchModuleLoader #431